build/ci-build.yml

```diff
@@ -0,0 +1,119 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - src/*
+
+pr:
+  paths:
+    include:
+      - src/*
+      - build/ci-build.yml
+
+variables:
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.402'
+  - name: 'Project'
+    value: 'Arcus.Template'
+  - name: 'Http.Port'
+    value: '5000'
+  - name: 'Arcus_Api_BaseUrl'
+    value: 'http://localhost:5000/api/v1'
```

Comment:
You should use the variable here as well

Reply:
Done.